frontend/src/employee-mobile-frontend/components/attendances/AttendanceDailyServiceTimes.tsx

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React from 'react'
+import { useTranslation } from '../../state/i18n'
+import { Translations } from '../../assets/i18n'
+import { DailyServiceTimes, TimeRange } from '../../api/attendances'
+import { ServiceTime } from './components'
+
+const dayNames = [
+  'monday',
+  'tuesday',
+  'wednesday',
+  'thursday',
+  'friday'
+] as const
+
+type DayName = typeof dayNames[number]
+
+function getToday(): DayName {
+  return dayNames[new Date().getDay()]
+}
```

Comment:
This might have a bug or two. First I'm wondering if it was so that getDay has 0 for Sunday, so there might be an off-by-one error. Secondly if the current day is saturday/sunday this can return undefined I guess. In those cases we should also just show that noServiceToday text.

( I see this PR is still draft, but already commenting as I don't have much of anything better to do right now. )

Reply:
Fixed!